generatelog

```diff
@@ -9,48 +9,56 @@ source $OPENLAB_TOPDIR/Include/bmcop.sh
 source $OPENLAB_TOPDIR/Include/board_serial_op.sh
 source $OPENLAB_TOPDIR/Include/board_power_op.sh
 source $OPENLAB_TOPDIR/Include/pduop.sh
-cat /dev/null > boards_used_log
-
-board_all_no=`grep -E "^(BOARD)" $OPENLAB_CONF_DIR/$BOARD_INFO_FILE | grep -Po "(?<=BOARD)([^:]*)" | tr '\n' " "`
 
-board_all_usr=$(cut -d ":" -f1 $OPENLAB_CONF_DIR/$USER_INFO_FILE | grep ^[a-zA-Z]|tr '\n' " ")
+:> boards_used_log
+echo -e "We start to test at `date`" >> boards_used_log
 
-echo -e "We start to test at `date`">>boards_used_log
+board_all_no=$(grep -E "^BOARD" $OPENLAB_CONF_DIR/$BOARD_INFO_FILE | grep -Po "(?<=BOARD)([^:]*)" | tr '\n' " ")
```

Comment:
How about considering add common functions in Include/boardop.sh to get all boards / all users?